sklearn/externals/_lobpcg.py

```diff
@@ -108,8 +108,8 @@ def _b_orthonormalize(B, blockVectorV, blockVectorBV=None, retInvR=False):
     VBV = np.matmul(blockVectorV.T.conj(), blockVectorBV)
     try:
         # VBV is a Cholesky factor from now on...
-        VBV = cholesky(VBV, overwrite_a=True)
-        VBV = inv(VBV, overwrite_a=True)
+        VBV = cholesky(VBV, overwrite_a=True, check_finite=False)
```

Comment:
Because we divide by `normalization` on line 99, I think this may be non finite (for same reasons as mentioned here https://github.com/scikit-learn/scikit-learn/pull/18850#discussion_r526344155)